ppr-ui/src/components/tables/mhr/SearchedResults.vue

```diff
@@ -369,8 +403,19 @@ export default defineComponent({
       } else return '-'
     }
 
+    const getOwnerCount = (item: ManufacturedHomeSearchResultIF): string => {
+      const count = item.activeCount + item.exemptCount + item.historicalCount
+      if (count > 1) return `(${count})`
+    }
     const getItemClass = (item: ManufacturedHomeSearchResultIF): string => {
-      return item.selected && !props.isReviewMode ? 'selected' : ''
+      var rowClass = ''
```

Comment:
Try and use let to keep your variables scope to it's wrapping { }. 
In this case, it's the same as var but it's best practices. 

var - scoped to the parent function
let - scoped to the wrapping { } 

More granular control over your variables scope. Js can do funny things when the global scopes becoming polluted with all sorts of variables and references 👍

Reply:
Thanks! Was recently wondering the difference between var and let, this helps a lot :)